sample/ApiHubTrigger-Batch/function.json

```diff
@@ -0,0 +1,17 @@
+{
+  "bindings": [
+    {
+      "type": "apiHubTrigger",
+      "connectionString": "%dropbox%",
```

Comment:
@mathewc Can you point me to a sample which has something similar to AzureWebJobsDropBox in the function.json file?